MapleServer2/Types/Trophy.cs

```diff
@@ -134,4 +134,25 @@ private static bool RewardTypeRequiresClaim(RewardType type)
             _ => false
         };
     }
+
+    public static bool IsInConditionRange(string trophyCondition, string condition)
```

Comment:
This method doesn't feel like it should live here IMHO. If it were testing against the property `GradeCondition` I'd say put it here. To me, it feels like it should live in either `TrophyMetaDataStorage` or some new class that wraps these trophies.

Reply:
moved it